network.tf

```diff
@@ -1,11 +1,11 @@
 # Virtual network connection from virtual hub
 # Create a hub connection
 resource "azurerm_virtual_hub_connection" "hub_connection" {
```

Comment:
Could the `azurerm_virtual_hub_connection` be implemented as a separate sub-module, or to nest it within the hub sub-module since this functionality operates at the individual hub level?

friendly reminder to include the moved block if it has been decided to move it as suggested above

Reply:
Yes. We have started creating submodules. But it is not complete yet. P2S and vnet-hub-conn are remaining. We are releasing phases, and it was not part of the first cycle.

Reply:
Will it be possible to move this `azurerm_virtual_hub_connection` feature to this release as this linking of hub to spokes is something that is done hand in hand with creation of a hub

Reply:
No worries. I will try to squeeze it in.